src/indexumd.ts

```diff
@@ -2,7 +2,7 @@ import Cotter from "./Cotter";
 import CotterEnum from "./enum";
 
 class CotterExport extends Cotter {
-  CotterEnum: CotterEnum;
+  static CotterEnum = CotterEnum;
```

Comment:
I don't understand, why not export `CotterEnum` on line 8 ? just export both `CotterEnum` and `CotterExport`

Reply:
Becos I think can only export 1 thing. So in the webpack.config.js, can define what is the export name. I set the name as `Cotter` that's why `CotterExport` is accessible as `Cotter` when using the `<script>` tag.